topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1859,6 +1859,97 @@ Now that your production-ready Galaxy is running, try registering a new user and
+* **Destinations** - This section lists the different locations or queues etc that Galaxy can send jobs to. Each one has a name and uses a *plugin* to communicate with that location. They can specify things like the number of cpu's, amount of RAM to be allocated etc for DRMAA locations. Usually one of the destinations is set to be the default.
+* **Tools** - This section lists the various tools that Galaxy would like to send to the non-default *destination*. Each line in this section pairs up a tool in Galaxy with a particular job *destination*. Then everytime Galaxy gets a job for that particular tool, it is always sent to that *destination*.
+
+The basic `job_conf.xml` file looks like this:
+
+{% raw %}
+```xml
+<job_conf>
+    <plugins workers="4">
+        <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner"/>
+    </plugins>
+    <destinations>
+        <destination id="local" runner="local"/>
+    </destinations>
+    <tools>
+    </tools>
+</job_conf>
+```
+
+The above `job_conf.xml` file defines a *plugin* and *destination* to allow Galaxy to run user jobs on the local computer (i.e. The computer that Galaxy is running on.)
+
+Firstly, the plugins section contains a plugin called "local" which is of type "runner" and then loads the python code module for supporting local jobs. Next the destinations section contains a destination called "local" using the runner "local". As this is the only destination specified, it is also the default. So now everytime a user clicks "Execute" on a tool form, Galaxy will run the corresponding job locally using the python code specified.
+
+There are a lot of other plugins available for Galaxy for using other resources such as docker containers, kubernetes hosts, Pulsar destinations and HPC clusters to name a few. See the Galaxy documentation on [job configuration](https://docs.galaxyproject.org/en/master/admin/jobs.html) for more details on these plugins and their configuration. There is also an advanced sample job conf file located at: `/srv/galaxy/server/lib/galaxy/config/sample/job_conf.xml.sample_advanced`
+
+{% endraw %}
+
+> ### {% icon hands_on %} Hands-on: Job Conf
+>
+> 1. If the folder does not exist, create `templates/galaxy/config` next to your `galaxy.yml` playbook (`mkdir -p templates/galaxy/config/`).
+>
+> 2. Create `templates/galaxy/config/job_conf.xml.j2` with the following contents:
+>
+>    ```xml
+>    <job_conf>
+>        <plugins workers="4">
+>            <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner"/>
+>        </plugins>
+>        <destinations default="local">
+>            <destination id="local" runner="local"/>
+>        </destinations>
+>        <tools>
+>        </tools>
+>    </job_conf>
+>    ```
+>
+> 3. Inform `galaxyproject.galaxy` of where you would like the `job_conf.xml` to reside in your group variables:
```

Comment:
```suggestion
> 3. Inform `galaxyproject.galaxy` of where you would like the `job_conf.xml` to reside, by setting it in your `group_vars/galaxyservers.yml`:
```

got this comment somewhere else that "it was too unclear" and ok, it should really be in their skill set to figure out if they're comprehension is good, but, maybe we'll give them a helping hand in the first session.